bizcharts error followed by cannot read property 'destroy' of undefined

<!--
Please use this template while reporting a bug and provide as much info as possible.
Questions should be posted to https://community.grafana.com
Use query inspector to troubleshoot issues: https://community.grafana.com/t/using-grafanas-query-inspector-to-troubleshoot-issues/2630
-->

**What happened**:
![2020-30-04_15-53-30](https://user-images.githubusercontent.com/623453/80766656-c2084380-8afa-11ea-9c6d-853035a45f79.png)

I have a stats panel. I clicked "Edit" on the panel. This works initially, then ~1 second in it switches to this bizcharts error message above.

Console shows:
```
react-dom.production.min.js:196 Error: max: -1 should not be less than min: 0
    at i.u.calculateTicks (BizCharts.js:7758)
    at i.u.initTicks (BizCharts.js:7778)
    at i.u.init (BizCharts.js:7721)
    at i.t (BizCharts.js:3270)
    at new i (BizCharts.js:7635)
    at Function.o.<computed> [as linear] (BizCharts.js:22215)
    at e.t.createScale (BizCharts.js:34006)
    at d.n.createScale (BizCharts.js:35359)
    at r.i._createScale (BizCharts.js:728)
    at r.i._initAttrs (BizCharts.js:1087)
```

A few seconds later, the chart goes black with error ` cannot read property 'destroy' of undefined`. 

Console log after it goes black:

```
react-dom.production.min.js:196 Error: max: -1 should not be less than min: 0
    at i.u.calculateTicks (BizCharts.js:7758)
    at i.u.initTicks (BizCharts.js:7778)
    at i.u.init (BizCharts.js:7721)
    at i.t (BizCharts.js:3270)
    at new i (BizCharts.js:7635)
    at Function.o.<computed> [as linear] (BizCharts.js:22215)
    at e.t.createScale (BizCharts.js:34006)
    at d.n.createScale (BizCharts.js:35359)
    at r.i._createScale (BizCharts.js:728)
    at r.i._initAttrs (BizCharts.js:1087)
fu @ vendors~app.0081fd61c194d2e8aa31.js:2
i.componentDidCatch.r.callback @ vendors~app.0081fd61c194d2e8aa31.js:2
Oi @ vendors~app.0081fd61c194d2e8aa31.js:2
mi @ vendors~app.0081fd61c194d2e8aa31.js:2
mc @ vendors~app.0081fd61c194d2e8aa31.js:2
KqkS.t.unstable_runWithPriority @ vendors~app.0081fd61c194d2e8aa31.js:2
Wo @ vendors~app.0081fd61c194d2e8aa31.js:2
gc @ vendors~app.0081fd61c194d2e8aa31.js:2
ic @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
KqkS.t.unstable_runWithPriority @ vendors~app.0081fd61c194d2e8aa31.js:2
Wo @ vendors~app.0081fd61c194d2e8aa31.js:2
Xo @ vendors~app.0081fd61c194d2e8aa31.js:2
Yo @ vendors~app.0081fd61c194d2e8aa31.js:2
ec @ vendors~app.0081fd61c194d2e8aa31.js:2
enqueueSetState @ vendors~app.0081fd61c194d2e8aa31.js:2
O.setState @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
next @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.nextInfiniteTimeWindow @ vendors~app.0081fd61c194d2e8aa31.js:2
next @ app.0081fd61c194d2e8aa31.js:2
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.notifyNext @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
complete @ vendors~app.0081fd61c194d2e8aa31.js:2
r @ vendors~app.0081fd61c194d2e8aa31.js:2
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
t @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
t.connect @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
a @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
a @ vendors~app.0081fd61c194d2e8aa31.js:2
t._innerSub @ vendors~app.0081fd61c194d2e8aa31.js:2
t._tryNext @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
forEach.e.<computed> @ vendors~app.0081fd61c194d2e8aa31.js:2
b @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
b @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
n.onRefresh @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
l.emit @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
forEach.e.<computed> @ vendors~app.0081fd61c194d2e8aa31.js:2
j @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
j @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
Show 75 more frames
react-dom.production.min.js:196 TypeError: Cannot read property 'destroy' of undefined
    at e.value (BizCharts.js:55847)
    at t.value (BizCharts.js:54482)
    at react-dom.production.min.js:196
    at yu (react-dom.production.min.js:199)
    at mu (react-dom.production.min.js:204)
    at mc (react-dom.production.min.js:249)
    at KqkS.t.unstable_runWithPriority (scheduler.production.min.js:18)
    at Wo (react-dom.production.min.js:120)
    at gc (react-dom.production.min.js:244)
    at ic (react-dom.production.min.js:223)
fu @ vendors~app.0081fd61c194d2e8aa31.js:2
i.componentDidCatch.r.callback @ vendors~app.0081fd61c194d2e8aa31.js:2
Oi @ vendors~app.0081fd61c194d2e8aa31.js:2
mi @ vendors~app.0081fd61c194d2e8aa31.js:2
mc @ vendors~app.0081fd61c194d2e8aa31.js:2
KqkS.t.unstable_runWithPriority @ vendors~app.0081fd61c194d2e8aa31.js:2
Wo @ vendors~app.0081fd61c194d2e8aa31.js:2
gc @ vendors~app.0081fd61c194d2e8aa31.js:2
ic @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
KqkS.t.unstable_runWithPriority @ vendors~app.0081fd61c194d2e8aa31.js:2
Wo @ vendors~app.0081fd61c194d2e8aa31.js:2
Xo @ vendors~app.0081fd61c194d2e8aa31.js:2
Yo @ vendors~app.0081fd61c194d2e8aa31.js:2
ec @ vendors~app.0081fd61c194d2e8aa31.js:2
enqueueSetState @ vendors~app.0081fd61c194d2e8aa31.js:2
O.setState @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
next @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.nextInfiniteTimeWindow @ vendors~app.0081fd61c194d2e8aa31.js:2
next @ app.0081fd61c194d2e8aa31.js:2
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.notifyNext @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
complete @ vendors~app.0081fd61c194d2e8aa31.js:2
r @ vendors~app.0081fd61c194d2e8aa31.js:2
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
t @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
t.connect @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
a @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
a @ vendors~app.0081fd61c194d2e8aa31.js:2
t._innerSub @ vendors~app.0081fd61c194d2e8aa31.js:2
t._tryNext @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
forEach.e.<computed> @ vendors~app.0081fd61c194d2e8aa31.js:2
b @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
b @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
n.onRefresh @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
l.emit @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
forEach.e.<computed> @ vendors~app.0081fd61c194d2e8aa31.js:2
j @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
j @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
Show 75 more frames
react-dom.production.min.js:196 Error: max: -1 should not be less than min: 0
    at i.u.calculateTicks (BizCharts.js:7758)
    at i.u.initTicks (BizCharts.js:7778)
    at i.u.init (BizCharts.js:7721)
    at i.t (BizCharts.js:3270)
    at new i (BizCharts.js:7635)
    at Function.o.<computed> [as linear] (BizCharts.js:22215)
    at e.t.createScale (BizCharts.js:34006)
    at d.n.createScale (BizCharts.js:35359)
    at r.i._createScale (BizCharts.js:728)
    at r.i._initAttrs (BizCharts.js:1087)
fu @ vendors~app.0081fd61c194d2e8aa31.js:2
i.componentDidCatch.r.callback @ vendors~app.0081fd61c194d2e8aa31.js:2
Oi @ vendors~app.0081fd61c194d2e8aa31.js:2
mi @ vendors~app.0081fd61c194d2e8aa31.js:2
mc @ vendors~app.0081fd61c194d2e8aa31.js:2
KqkS.t.unstable_runWithPriority @ vendors~app.0081fd61c194d2e8aa31.js:2
Wo @ vendors~app.0081fd61c194d2e8aa31.js:2
gc @ vendors~app.0081fd61c194d2e8aa31.js:2
ic @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
KqkS.t.unstable_runWithPriority @ vendors~app.0081fd61c194d2e8aa31.js:2
Wo @ vendors~app.0081fd61c194d2e8aa31.js:2
Xo @ vendors~app.0081fd61c194d2e8aa31.js:2
Yo @ vendors~app.0081fd61c194d2e8aa31.js:2
ec @ vendors~app.0081fd61c194d2e8aa31.js:2
enqueueSetState @ vendors~app.0081fd61c194d2e8aa31.js:2
O.setState @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
next @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.nextInfiniteTimeWindow @ vendors~app.0081fd61c194d2e8aa31.js:2
next @ app.0081fd61c194d2e8aa31.js:2
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.notifyNext @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
complete @ vendors~app.0081fd61c194d2e8aa31.js:2
r @ vendors~app.0081fd61c194d2e8aa31.js:2
t.__tryOrUnsub @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t._complete @ vendors~app.0081fd61c194d2e8aa31.js:2
t.complete @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
t @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
t.connect @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
a @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
a @ vendors~app.0081fd61c194d2e8aa31.js:2
t._innerSub @ vendors~app.0081fd61c194d2e8aa31.js:2
t._tryNext @ vendors~app.0081fd61c194d2e8aa31.js:2
t._next @ vendors~app.0081fd61c194d2e8aa31.js:2
t.next @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
e._trySubscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
e.call @ vendors~app.0081fd61c194d2e8aa31.js:2
e.subscribe @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
forEach.e.<computed> @ vendors~app.0081fd61c194d2e8aa31.js:2
b @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
b @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
n.onRefresh @ default~DashboardPage~SoloPanelPage.0081fd61c194d2e8aa31.js:1
l.emit @ vendors~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
c @ vendors~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
forEach.e.<computed> @ vendors~app.0081fd61c194d2e8aa31.js:2
j @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
Promise.then (async)
j @ app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ vendors~app.0081fd61c194d2e8aa31.js:2
i @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
setTimeout (async)
i.defer @ angular~app.0081fd61c194d2e8aa31.js:2
a @ angular~app.0081fd61c194d2e8aa31.js:2
value @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
p @ angular~app.0081fd61c194d2e8aa31.js:2
(anonymous) @ angular~app.0081fd61c194d2e8aa31.js:2
Show 75 more frames
react-dom.production.min.js:196 TypeError: Cannot read property 'destroy' of undefined
    at e.value (BizCharts.js:55847)
    at t.value (BizCharts.js:54482)
    at react-dom.production.min.js:196
    at yu (react-dom.production.min.js:199)
    at mu (react-dom.production.min.js:204)
    at mc (react-dom.production.min.js:249)
    at KqkS.t.unstable_runWithPriority (scheduler.production.min.js:18)
    at Wo (react-dom.production.min.js:120)
    at gc (react-dom.production.min.js:244)
    at ic (react-dom.production.min.js:223)
```


Panel json:

<details>

```json
{
  "datasource": "Prometheus",
  "cacheTimeout": null,
  "gridPos": {
    "h": 3,
    "w": 6,
    "x": 6,
    "y": 3
  },
  "id": 23763571993,
  "links": [],
  "targets": [
    {
      "expr": "sum(rate(istio_requests_total{reporter=\"destination\", response_code!~\"5.*\"}[1m])) / sum(rate(istio_requests_total{reporter=\"destination\"}[1m]))",
      "format": "time_series",
      "intervalFactor": 1,
      "refId": "A",
      "step": 4
    }
  ],
  "title": "Global Success Rate (non-5xx responses)",
  "type": "stat",
  "transparent": true,
  "options": {
    "reduceOptions": {
      "values": false,
      "calcs": [
        "mean"
      ]
    },
    "orientation": "horizontal",
    "colorMode": "value",
    "graphMode": "area",
    "justifyMode": "center",
    "fieldOptions": {
      "calcs": [
        "mean"
      ]
    }
  },
  "fieldConfig": {
    "defaults": {
      "custom": {},
      "unit": "percentunit",
      "max": -1,
      "decimals": 2,
      "title": "",
      "noValue": "",
      "thresholds": {
        "mode": "absolute",
        "steps": [
          {
            "value": null,
            "color": "rgba(245, 54, 54, 0.9)"
          },
          {
            "value": 95,
            "color": "rgba(237, 129, 40, 0.89)"
          },
          {
            "value": 99,
            "color": "rgba(50, 172, 45, 0.97)"
          }
        ]
      },
      "mappings": [
        {
          "op": "=",
          "text": "N/A",
          "value": "",
          "id": 0,
          "type": 1
        }
      ],
      "nullValueMode": "connected"
    },
    "overrides": []
  },
  "pluginVersion": "7.0.0-beta1",
  "timeFrom": null,
  "timeShift": null
}
```

</details>

**What you expected to happen**:

No error

**How to reproduce it (as minimally and precisely as possible)**:

I am able to reproduce with the above panel and selecting edit

**Anything else we need to know?**:

**Environment**:

Grafana version: 7.0.0-beta1
Data source type & version: Prometheus
OS Grafana is installed on: docker image with tag 7.0.0-beta1 on Kubernetes
User OS & Browser: Linux Chrome 81
Grafana plugins:
Others:

--- a/packages/grafana-data/src/field/fieldOverrides.test.ts
+++ b/packages/grafana-data/src/field/fieldOverrides.test.ts
@@ -11,14 +11,14 @@ import { MutableDataFrame, toDataFrame } from '../dataframe';
 import {
   DataFrame,
   Field,
+  FieldColorModeId,
   FieldConfig,
   FieldConfigPropertyItem,
   FieldConfigSource,
   FieldType,
   InterpolateFunction,
-  ThresholdsMode,
-  FieldColorModeId,
   ScopedVars,
+  ThresholdsMode,
 } from '../types';
 import { locationUtil, Registry } from '../utils';
 import { mockStandardProperties } from '../utils/tests/mockStandardProperties';
@@ -86,6 +86,21 @@ describe('Global MinMax', () => {
     const minmax = findNumericFieldMinMax([f0]);
     expect(minmax.min).toEqual(-20);
     expect(minmax.max).toEqual(1234);
+  });
+
+  describe('when value is null', () => {
+    it('then global min max should be null', () => {
+      const frame = toDataFrame({
+        fields: [
+          { name: 'Time', type: FieldType.time, values: [1] },
+          { name: 'Value', type: FieldType.number, values: [null] },
+        ],
+      });
+      const { min, max } = findNumericFieldMinMax([frame]);
+
+      expect(min).toBeNull();
+      expect(max).toBeNull();
+    });
   });
 });
 

--- a/packages/grafana-data/src/field/fieldOverrides.ts
+++ b/packages/grafana-data/src/field/fieldOverrides.ts
@@ -55,11 +55,23 @@ export function findNumericFieldMinMax(data: DataFrame[]): GlobalMinMax {
     for (const field of frame.fields) {
       if (field.type === FieldType.number) {
         const stats = reduceField({ field, reducers });
-        if (stats[ReducerID.min] < min) {
-          min = stats[ReducerID.min];
+        const statsMin = stats[ReducerID.min];
+        const statsMax = stats[ReducerID.max];
+
+        if (!statsMin) {
+          min = statsMin;
         }
-        if (stats[ReducerID.max] > max) {
-          max = stats[ReducerID.max];
+
+        if (!statsMax) {
+          max = statsMax;
+        }
+
+        if (statsMin && statsMin < min) {
+          min = statsMin;
+        }
+
+        if (statsMax && statsMax > max) {
+          max = statsMax;
         }
       }
     }

